server/bin/unittests

```diff
@@ -253,6 +253,15 @@ function _verify_output {
     return ${res}
 }
 
+function move_out_of_maint {
+    # Given an mock "/var/www/html", move the target Pbench Server out of
```

Comment:
"a" mock... although calling out "mock" at all seems a bit heavy. E.g., this function would work perfectly well on the real `/var/www/html` or any other directory with the expected subdirectory structure. 🤔